packages/slice-machine/src/components/HoverCard/HoverCard.test.tsx

```diff
@@ -1,24 +1,129 @@
 // @vitest-environment jsdom
+import React from "react";
 import { describe, test, expect, vi } from "vitest";
-import { render, screen } from "../../../test/__testutils__";
-import { HoverCard } from "./HoverCard";
 
-// TBD: it's odd that this needs to be included in this test
+import { screen, render, waitFor } from "../../../test/__testutils__"; // skip usinng this
```

Comment:
Which one?

Reply:
Which one what?  🤔

It's your comment 😛

I will remove it, for me, we should not add comment like this we never know why it's for after :/

<img width="1191" alt="Screenshot 2023-07-25 at 16 20 30" src="https://github.com/prismicio/slice-machine/assets/19946868/7ff33c51-93a8-4337-a2a1-959b873be1c1">

Reply:
Removed 🙂

Reply:
oh, I remember now... it's because the component doesn't need to have the initial state setup :)

Reply:
I see, but it's not a big problem, no?